README.md

```diff
@@ -18,67 +18,112 @@ This sample also doesn't use any help to build the JSON strings for the OMF mess
 
 [OMF documentation](https://omf-docs.osisoft.com/)
 
-## Getting Started
+## To Run this Sample in Visual Studio:
 
-In this example we assume that you have the dotnet core CLI.
+1. Clone the GitHub repository
+2. Open the solution file in Microsoft Visual Studio, [OMF_API.sln](OMF_API.sln)
+3. Rename the file [appsettings.placeholder.json](OMF_API/appsettings.placeholder.json) to appsettings.json
+4. Update appsettings.json with the credentials for the enpoint(s) you want to send to. See [Configure endpoints and authentication](#configure-endpoints-and-authentication) below for additional details
+5. Click **Debug** > **Start Debugging** (or F5)
 
-To run this example from the commandline run
+## To Test this Sample in Visual Studio:
 
-```shell
-dotnet restore
-dotnet run
-```
+1. Follow steps 1-4 from the section above
+2. Click **Test** > **Run All Tests** (or Ctrl+R, A)
 
-to test this program change directories to the test and run
+## Customizing the Application
 
-```shell
-dotnet restore
-dotnet test
-```
+This application can be customized to send your own custom types, containers, and data by modifying the [OMF-Types.json](OMF_API/OMF-Types.json), 
+[OMF-Containers.json](OMF_API/OMF-Containers.json), and [OMF-Data.json](OMF_API/OMF-Data.json) files respectively. Each one of these files contains an array of OMF json objects, which are
+created in the endpoints specified in [config.json](OMF_API/config-placeholder.json) when the application is run. For more information on forming OMF messages, please refer to our 
+[OMF version 1.1 documentation](https://omf-docs.osisoft.com/documentation_v11/Whats_New.html).  
+  
+In addition to modifying the json files mentioned above, the get_data function in [program.py](OMF_API/program.py) should be updated to populate the OMF data messages specified in 
+[OMF-Data.json](OMF_API/OMF-Data.json) with data from your data source.  
+  
+Finally, if there are any other activities that you would like to be running continuously, this logic can be added under the while loop in the main() function of 
+[program.py](OMF_API/program.py).
 
-## Configure constants for connecting and authentication
+## Configure Endpoints and Authentication
 
-The sample is configured using the file [appsettings.placeholder.json](appsettings.placeholder.json). Before editing, rename this file to `appsettings.json`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
+The sample is configured using the file [config-placeholder.json](OMF_API/config-placeholder.json). Before editing, rename this file to `config.json`. This repository's `.gitignore` rules should prevent the file from ever being checked in to any fork or branch, to ensure credentials are not compromised.
```

Comment:
I will change the README to refer to the appsettings.placeholder.json file

Reply:
I get now why you named it config, since you were trying to use the same file for both samples. Whenever we get back to that item to use the same config file for all the samples we might need to think about this again. For now thanks for making it consistent with the other dotnet samples.